### PR TITLE
FLW-54 assamese language translation

### PR DESCRIPTION
**Pull request by ritik-bizbrolly**
Added translation for assamese languages.

### CODE REVIEW

**Review by drtechie — CHANGES_REQUESTED** [review]
There are conflcts in the PR.
Several files still show English instead of Assamese.

File: app/src/main/res/values-hi/strings_pwr.xml
Comment (drtechie) [inline app/src/main/res/values-hi/strings_pwr.xml:1]: @ritik-bizbrolly was referring to this one regarding English.

Comment (drtechie) [reply app/src/main/res/values-hi/strings_pwr.xml:1]: This is under hi folder.
So should be Hindi right?

Comment (sushant-bizbrolly) [reply app/src/main/res/values-hi/strings_pwr.xml:1]: These are Hindi translations, I asked Bala to add a ticket for Hindi translations. Will be working on it once everything is ready.

**Comment by sushant-bizbrolly** [comment]
Conflicts will be resolved once we merge the code.

**Pull request merged**